code/modules/reagents/chemistry/reagents/food_reagents.dm

```diff
@@ -136,7 +136,7 @@
 	return TRUE
 
 /datum/reagent/consumable/cooking_oil/reaction_turf(turf/open/T, reac_volume)
-	if(!istype(T))
+	if(!istype(T) || isgroundlessturf(T)))
```

Comment:
remove one here you doofus

Reply:
Wtf goof hacked my code and put that there I swear